control/setup.py

```diff
@@ -1,248 +1,198 @@
-from setuptools import setup, find_packages, Extension
-from setuptools.command.test import test as TestCommand
+import os
+import shlex
+import subprocess
+import sys
+from distutils import log
 from distutils.command.build_ext import build_ext
 from distutils.command.clean import clean
-from distutils import log
 from distutils.errors import LibError
-import versioneer
 
-import os
-import glob
-import sys
-import subprocess
-import shlex
+from setuptools import Extension, setup
+
+sys.path.append(".")  # pipenv install fails without this - pip install works fine
+import versioneer
 
 # Extract the stub-only argument from the command line if present
 stub_only = False
-if '--stub-only' in sys.argv:
+if "--stub-only" in sys.argv:
     stub_only = True
-    sys.argv.remove('--stub-only')
-    
-# Define the stub fem_api extension modules 
-fem_api_extension_path='fem_api_extension'
-fem_api_wrapper_source = os.path.join(fem_api_extension_path, 'fem_api_wrapper.c')
+    sys.argv.remove("--stub-only")
+
+# Define the stub fem_api extension modules
+fem_api_extension_path = "src/fem_api_extension"
+fem_api_wrapper_source = os.path.join(fem_api_extension_path, "fem_api_wrapper.c")
 
-fem_api_stub_source_path=os.path.join(fem_api_extension_path, 'api_stub')
+fem_api_stub_source_path = os.path.join(fem_api_extension_path, "api_stub")
 fem_api_stub_sources = [fem_api_wrapper_source] + [
-                            os.path.join(fem_api_stub_source_path, source) for source in [
-                                'femApi.cpp', 'ExcaliburFemClient.cpp', 'FemApiError.cpp']
-                             ]
+    os.path.join(fem_api_stub_source_path, source)
+    for source in ["femApi.cpp", "ExcaliburFemClient.cpp", "FemApiError.cpp"]
+]
 
-fem_api_stub = Extension('excalibur.fem_api_stub', 
+fem_api_stub = Extension(
+    "excalibur_control.fem_api_stub",
     sources=fem_api_stub_sources,
     include_dirs=[fem_api_stub_source_path],
-    define_macros=[('COMPILE_AS_STUB', None)],
+    define_macros=[("COMPILE_AS_STUB", None)],
 )
 
 # Add the stub to the list of extension modules to build
 fem_ext_modules = [fem_api_stub]
 
 # If the stub_only option is not set, define the full fem_api extension module and add to the list
 if not stub_only:
-    
-    fem_api_path=os.path.join(fem_api_extension_path, 'api')
-    fem_api_source_path=os.path.join(fem_api_path, 'src')
-    fem_api_include_path=os.path.join(fem_api_path, 'include')
-    
-    fem_api_sources = [fem_api_wrapper_source] 
-    
-    fem_api = Extension('excalibur.fem_api',
+
+    fem_api_path = os.path.join(fem_api_extension_path, "api")
+    fem_api_source_path = os.path.join(fem_api_path, "src")
+    fem_api_include_path = os.path.join(fem_api_path, "include")
+
```

Comment:
Lots of changes here which are just style alterations.  Are these all due to running a tool against the code (Black)?

Reply:
Argh, I did originally do a separate commit of just running black, but I must have squashed it. Yes the majority of this diff is just black formatting. I will try to unpick that again.